docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,65 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+For experienced developers, here is a summary of the guide, see each section for more details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) follows instructions on their site for your OS.
+
+- Install WordPress environment tool
+
+```
+npm -g install @wordpress/env
+```
+
```

Comment:
Is it good with some info on what wordpress/env is.

Reply:
Result:

```
paaljoachimromdahl@Paals-MacBook-Pro ~ % npm -g install @wordpress/env
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
/Users/paaljoachimromdahl/.nvm/versions/node/v12.18.3/bin/wp-env -> /Users/paaljoachimromdahl/.nvm/versions/node/v12.18.3/lib/node_modules/@wordpress/env/bin/wp-env

> nodegit@0.26.5 install /Users/paaljoachimromdahl/.nvm/versions/node/v12.18.3/lib/node_modules/@wordpress/env/node_modules/nodegit
> node lifecycleScripts/preinstall && node lifecycleScripts/install

[nodegit] Running pre-install script
[nodegit] Running install script
node-pre-gyp
WARN Using request for node-pre-gyp https download
[nodegit] Success: "/Users/paaljoachimromdahl/.nvm/versions/node/v12.18.3/lib/node_modules/@wordpress/env/node_modules/nodegit/build/Release/nodegit.node" is installed via remote
[nodegit] Completed installation successfully.

> nodegit@0.26.5 postinstall /Users/paaljoachimromdahl/.nvm/versions/node/v12.18.3/lib/node_modules/@wordpress/env/node_modules/nodegit
> node lifecycleScripts/postinstall

+ @wordpress/env@1.6.0
added 274 packages from 236 contributors in 82.855s
paaljoachimromdahl@Paals-MacBook-Pro ~ % 

```

Reply:
This is in the quick start, if they want details they can read below that explains.